tests/Application/.env

```diff
@@ -24,3 +24,8 @@ DATABASE_URL=mysql://root@127.0.0.1/sylius?serverVersion=mariadb-10.5.5
 # Delivery is disabled by default via "null://localhost"
 MAILER_URL=smtp://localhost:1025
 ###< symfony/swiftmailer-bundle ###
+
+###> monsieurbiz/sylius-anti-spam-plugin ###
+GOOGLE_RECAPTCHA_SITE_KEY=6LfgYWcaAAAAACcBJxYvaaWZQ7A1mYB7kxs0Y4fb
+GOOGLE_RECAPTCHA_SECRET=6LfgYWcaAAAAAHEl0A-C9i2XY9IPjMtmXYAAQ9Qj
```

Comment:
Where do they come from?

Reply:
<img width="1077" alt="image" src="https://user-images.githubusercontent.com/11380627/109159852-638d0f80-7775-11eb-8d07-7b7fe92b9b45.png">

Reply:
They are removed from repo and the key is deleted